products/cos.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Google Container-Optimized OS (COS)
+layout: post
+category: os
+changelogTemplate: "https://cloud.google.com/container-optimized-os/docs/release-notes/__RELEASE_CYCLE__#__LATEST__"
+iconSlug: googlecloud
+permalink: /cos
+alternate_urls:
+  - /google-cos
+  - /container-optimized-os
+releasePolicyLink: https://cloud.google.com/container-optimized-os/docs/resources/support-policy
+releaseColumn: true
+releaseDateColumn: true
+activeSupportColumn: Active Milestones
+eolColumn: Deprecated Milestones
+versionCommand: cat /etc/os-release /etc/lsb-release
+
+releases:
+  # LTS Milestones
+  - releaseCycle: "m101"
```

Comment:
Let's keep releaseCycle as integers (`releaseCycle: "101"`) with the `releaseLabel` set globally for the `m` prefix. (`releaseLabel: m__RELEASE_CYCLE__`)